components/src/Input/Input.js

```diff
@@ -51,6 +52,12 @@ const StyledInput = styled.input(
     "::placeholder": {
       color: transparentize(0.4, theme.colors.black),
     },
+    "&[type='search']": {
```

Comment:
Could this be put directly on the NavBarSearch component instead of on the Input? Keeps the input component a bit cleaner and keeps relevant stylings directly on the component we want to apply it to.

Reply:
I was debating where this should go and originally had it in navbar search component but then it seemed like more of a reset thing that should be pushed level up. But I agree it would be less confusing if it was in navbar search component so I'll move it there.